.github/workflows/py-build-main.yml

```diff
@@ -99,7 +99,7 @@ jobs:
         with:
           submodules: true
 
-      - uses: pypa/cibuildwheel@v2.17.0
+      - uses: pypa/cibuildwheel@v2.19.2
```

Comment:
This version bump fixes build breakage related to end-of-life of CentOS 7 (July 1st) which was used in the cibuildwheel automated build tool.